addons/api/mirage/serializers/managed-groups.js

```diff
@@ -0,0 +1,13 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  modelName: 'managed-groups',
+
+  _hashForModel() {
+    const json = ApplicationSerializer.prototype._hashForModel.apply(
+      this,
+      arguments
+    );
+    return json;
```

Comment:
@randallmorey 
Should I add here something like `json.auth_method_id=model.authMethodId`?